src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,39 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	private static final int ARRAY_SIZE = 12;
+	private static final int EMPTY_ARRAY = -1;
+	private int total = -1;
+	public int getTotal() {
+		return total;
+	}
+
+	final private int[] numbers = new int[ARRAY_SIZE];
+
+	public void countIn(int in) {
+		if (!isFull())
+			numbers[++total] = in;
+	}
+
+	public boolean callCheck() {
+		return total == EMPTY_ARRAY;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you asked for review hints (1.4.w7): tiny.cc/998muz ?
This constant should be different (separate) than the one in peekaboo and countOut methods.